src/chainer.js

```
@@ -1,4 +1,4 @@
-'use strict'
+'use strict';
```

Optional suggestion: consider validating the `functions` parameter and documenting the unary requirement. For example, you might check that `functions` is an array and that every element is a function, and decide whether to throw an error for invalid inputs or silently ignore them. This improves robustness and makes the function's expectations explicit for callers (not required by the task/tests).